src/ipsis/course-merge/SelectedCourseMergeOfferingCollectionEntity.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * SelectedCourseMergeOfferingCollectionEntity class representation of a list of course merge offering as defined in the LMS
+ * See: ISirenCourseMergeSerializer.SerializeSelectedCourseOfferingListResult
+ */
+import { Actions } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class SelectedCourseMergeOfferingCollectionEntity extends Entity {
+	courseMergeOfferings() {
+		if (!this._entity) {
+			return;
+		}
+		return this._entity.entities;
+	}
+
+	selectedCourseMergeOffering() {
+		return this._entity && this._entity.getSubEntitiesByClass('target');
+	}
+
+	hasMergeAction() {
+		return this._entity.hasActionByName(Actions.ipsis.sisCourseMerge.mergeCourseOfferings);
```

Comment:
Should this be `_entity?`, or do we know it's non-null because something must be selected to have this action available?

Reply:
`_entity` is set by the constructor of the base class. It should always be non-null.